sdk/appconfig/azappconfig/client_test.go

```diff
@@ -15,10 +15,16 @@ import (
 )
 
 func TestClient(t *testing.T) {
+	connectionString := os.Getenv("APPCONFIGURATION_CONNECTION_STRING")
+	if connectionString == "" {
+		// TODO: Need CI to deploy test-resources.json
```

Comment:
Note: you can use the python test-resources.json file [here](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/appconfiguration/test-resources.json)

Reply:
test-resources.json is already checked in, the problem is that CI is not deploying it. I run tests manually at this point.

Reply:
@antkmsft I see the template getting deployed here? https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1417572&view=logs&j=a46fb01e-fa54-52e9-355f-ff40079f227e&t=5d7559a5-142f-5cf9-0231-17146bcf4d47

Reply:
Ah, I see, it does run live and pass in tests, it fails during regular unit tests. I'll try to see if I can mark it as live-only.

Reply:
Currently it will run during the live test, when the template is deployed and the corresponding environment variable is set.

Reply:
Live tests do run and pass now: https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1419414&view=logs&j=a46fb01e-fa54-52e9-355f-ff40079f227e&t=db30c526-9df4-58f3-c8d1-1445458d3e15